app/uk/gov/hmrc/eusubsidycompliance/controllers/MicroserviceHelloWorldController.scala

```diff
@@ -25,7 +25,7 @@ import scala.concurrent.Future
 class MicroserviceHelloWorldController @Inject()(cc: ControllerComponents)
     extends BackendController(cc) {
 
-  def hello(): Action[AnyContent] = Action.async { implicit request =>
+  def hello(): Action[AnyContent] = Action.async { _ =>
```

Comment:
Shall we remove this?

Reply:
It's being removed in the next PR. This just deals with the immediate warnings for now.